sinch/core/deserializers.py

```diff
@@ -2,4 +2,4 @@
 
 
 def timestamp_to_datetime_in_utc_deserializer(timestamp: str):
-    return datetime.fromisoformat(timestamp + 'Z')
+    return datetime.fromisoformat(timestamp + "+00:00")
```

Comment:
What happens with all the other dates in the other SDKs which are returned with `Z`? Does it work with older versions of Python (which ones?)?

Reply:
`"+00:00"` works for all versions of Python I have to currently support.
`Z` only works in CPython 3.11 and 3.12 (std lib change)

Reply:
This discussion is way beyond this PR :) We can discuss this on architecture meeting.
I'll keep my implementation for now (current API).

Reply:
I've seen a ticket regarding timestamp for verification API. I believe you'll tackle all the APIs to support the dates